CONTRIBUTING.md

```diff
@@ -1,11 +1,15 @@
+Thanks for your interest in Juju! Contributions like yours make good projects
+great.
+
 Contents
 ========
 
-1. [Getting started](#getting-started)
-2. [Dependency management](#dependency-management)
-3. [Code formatting](#code-formatting)
-4. [Workflow](#workflow)
-5. [Community](#community)
+1. [Building Juju](#building-juju)
+2. [Getting started](#getting-started)
```

Comment:
Looks like we have lost this heading now.

Reply:
Good catch, fixed.